airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/PostgresIntegrationTest.java

```diff
@@ -28,7 +28,6 @@
 import com.google.common.collect.ImmutableMap;
 import io.airbyte.commons.json.Jsons;
 import io.airbyte.db.Databases;
-import io.airbyte.integrations.base.NamingHelper;
```

Comment:
should this be in this PR? are tests not passing?

Reply:
accidental, removed